### ISSUE TITLE
what is your system environment?Can you give me some more ditails?

### ISSUE THREAD

**Issue by Edwardmark** [Jack-CV/PCN#9]
which cuda version, cudnn version and lots of dependency version are you using? Can you give us some instructions to test your demo? Thank you very much.

**Comment by Edwardmark** [comment]
compile picture
libPCN.so: undefined reference to `caffe::Net<float>::Net(std::string const&, caffe::Phase, int, std::vector<std::string, std::allocator<std::string> > const*)'
collect2: error: ld returned 1 exit status
done
I always met this error, anybody knows why?

**Comment by Edwardmark** [comment]
@jack-cv, finally I compile your code successfully, I got:  
`
root@b50c61f18b38:/app/code# sh compile.sh picture
compile.sh: 1: compile.sh: i#!/usr/bin/en: not found
rm picture
compile picture
done
`
but when I run it, I got the following error: 
`root@b50c61f18b38:/app/code# sh run.sh picture
./picture: error while loading shared libraries: libPCN.so: cannot open shared object file: No such file or directory
`  
when I run ls cmd, I got:
`root@b50c61f18b38:/app/code# ls
PCN.dat  PCN.h  compile.sh  glog-0.3.3  imgs  libPCN.so  picture  picture.cpp  run.sh  v0.3.3.tar.gz  video.cpp`
We can see that libPCN.so is right there, why I cannot run picture? Any advise?
Thank you very much.
